composer.json

```diff
@@ -6,12 +6,12 @@
     "license": "GPL-2.0-or-later",
     "minimum-stability": "dev",
     "require": {
-        "drupal/date_recur": "^3.0",
-        "drupal/date_recur_modular": "^3.0",
+        "drupal/date_recur": "^3.2",
+        "drupal/date_recur_modular": "3.x-dev#6ea45215",
```

Comment:
Can already go 3.1.0-rc1 https://www.drupal.org/project/date_recur_modular/releases/3.1.0-rc1

Reply:
Updated the reference - thanks